include/sender_cadmium.hpp

```diff
@@ -172,11 +228,21 @@ class Sender {
             return bags;
         }
 
-        // time_advance function
+        /**
+	* Function that returns the next internal transition time.
```

Comment:
You may consider saying 'function (with no parameter) that returns the ...'

Reply:
Done